api/app/Models/WorkExperience.php

```diff
@@ -62,6 +63,13 @@ class WorkExperience extends Experience
         'experience_type' => WorkExperience::class,
     ];
 
+    /**
+     * Listeners for model events
+     */
+    protected $dispatchesEvents = [
```

Comment:
I think we need to add the deleted event as well.  If someone deletes a work experience that could change what the "current value" is.

Reply:
oh yeah, that totally makes sense! Added in [add work experience deleted event to dispatch](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12670/commits/bce30cbc2937af21bbdf669af877fbd857a880cb)